web/src/components/storage/VolumeForm.jsx

```diff
@@ -721,13 +722,14 @@ export default function VolumeForm({ id, volume: currentVolume, templates = [],
   return (
     <Form id={id} onSubmit={submitForm}>
       <FormGroup isRequired label={_("Mount point")} fieldId="mountPoint">
-        <MountPointFormSelect
+        {currentVolume !== undefined} && <p> {state.formData.mountPoint} </p>
+        {/* <MountPointFormSelect
```

Comment:
Actually, the `MountPointFormSelect` should be mounted when the _currentVolume_ is defined. Check the comment above. 

But in any case, please do not comment dead code, just remove it :wink: